### PR TITLE
Md_Minhaj_Uddin_ICDCP_49

### PR DESCRIPTION
**Pull request by Minhaj28**
Add a common readme file for edit group member and create own folder

### CODE REVIEW

**Review by Minhaj28 — COMMENTED** [review]
nice work

**Review by Minhaj28 — COMMENTED** [review]
review all files

File: Minhaj_Uddin_ICDCP_49/README.md
Comment (Minhaj28) [inline Minhaj_Uddin_ICDCP_49/README.md:1]: ok

File: README.md
Comment (Minhaj28) [inline README.md:1]: ok

**Review by Minhaj28 — COMMENTED** [review]
ok review per file

**Comment by hmrafid** [comment]
reviewed

**Review by hmrafid — APPROVED** [review]
reviewed and approving merge

**Pull request merged**